client/app/domain/general-informations/domain-general-informations.controller.js

```diff
@@ -150,6 +150,13 @@ angular.module('App').controller(
           }&selectedType=DOMAIN`,
           isAvailable: () => true,
         },
+        manageAlldom: {
+          text: this.$translate.instant('common_manage'),
```

Comment:
It sounds a bit strange to have the same text as [`manageAutorenew.text` L147](https://github.com/ovh-ux/ovh-manager-web/pull/990/files#diff-173bd65a8e102b59a4e7f07cda4a8979R147).

Furthermore you are only using the `href` attribut so you can get rid of both `text` and `isAvailable` properties.

Reply:
Fixed